docs/dev/table/sourceSinks.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Table Sources & Sinks"
+title: "User-defined Sources & Sinks"
```

Comment:
User-defined **Table** Sources and Sinks?

Reply:
This would cause a line break in the navigation :/